### DISCUSSION TITLE
arm版本集群部署Milvus-2.4.9

### DISCUSSION THREAD

**Discussion by jhyever** [milvus-io/milvus#38728] [Announcement]
我看这个已经started了，为啥还是error呀
![image](https://github.com/user-attachments/assets/8da88f9c-a477-4b4b-a37b-1596d85352c3)
![image](https://github.com/user-attachments/assets/a3568065-8e99-4b6f-a800-ebba75653f10)

![image](https://github.com/user-attachments/assets/0c06f1da-9952-4a27-b359-332cfdb06caf)

**Comment by LoveEachDay** [comment]
@jhyever pulsar 2.9.5 doesn't support arm. You should try pulsar 3.0.x instead.

**Reply by jhyever** [reply]
![image](https://github.com/user-attachments/assets/3963ebda-a06d-439a-9a44-65db0cf245a1)
3.0.x的arm版本的也有呢，我试了一下也不行呢，还是这样
